app/controllers/DesktopLogin.scala

```diff
@@ -64,7 +64,7 @@ class DesktopLogin(
 
 
       if (validateUser(authedUserData)) {
-        val token = CookieUtils.generateCookieData(authedUserData, panDomainSettings.settings.signingKeyPair.getPrivate)
+        val token = CookieUtils.generateCookieData(authedUserData, panDomainSettings.settings.signingAndVerification)
```

Comment:
https://github.com/guardian/pan-domain-authentication/pull/150 means that code shouldn't directly reference private or public keys anymore (eg do not reference `settings.signingKeyPair`- there may be more accepted public keys than that field would have told you about!). Instead, we need to use `settings.signingAndVerification` or `publicSettings.verification`.